lib/nermesterts/game_picker.ex

```diff
@@ -26,6 +26,8 @@ defmodule GamePicker do
       nil
   """
   def pick_game(games_list, num_players) do
+    :rand.seed(:exsplus, DateTime.utc_now |> DateTime.to_date |> Date.to_erl)
```

Comment:
Looking at something like this:

```elixir
defmodule Randomizer do
  def new do
    :rand.seed(:exsplus, DateTime.utc_now |> DateTime.to_date |> Date.to_erl)
  end

  def new(seed) do
    :rand.seed(:exsplus, seed)
  end
end
```

Is that how the randomizer would work with injection?